src/libs/ajax/data-table-providers/WdsDataTableProvider.ts

```diff
@@ -86,7 +86,7 @@ const getRelationParts = (val: unknown): string[] => {
 export const getWdsUrl = apps => {
   // look explicitly for an app named 'cbas-wds-default'. If found, use it, even if it isn't running
   // this handles the case where the user has explicitly shut down the app
-  const namedApp = apps.filter(app => app.appType === 'CROMWELL' && app.appName === 'cbas-wds-default')
+  const namedApp = apps.filter(app => app.appType === 'CROMWELL' && app.appName === 'cbas-wds-default' && app.status === 'RUNNING')
```

Comment:
The appName should stay in concert with the name changes in https://github.com/DataBiosphere/terra-ui/pull/3673

Reply:
Agreed, one of us will have to rebase-